jobs/src/main/scala/dcos/metronome/model/JobRunSpec.scala

```diff
@@ -37,5 +42,21 @@ object JobRunSpec {
   val DefaultVolumes = Seq.empty[Volume]
   val DefaultRestartSpec = RestartSpec()
   val DefaultTaskKillGracePeriodSeconds = None
+
+  implicit lazy val validJobRunSpec: Validator[JobRunSpec] = new Validator[JobRunSpec] {
+    import com.wix.accord._
+    import ViolationBuilder._
+
+    override def apply(jobRunSpec: JobRunSpec): Result = {
+      if (jobRunSpec.cmd.isDefined || jobRunSpec.docker.exists(d => d.image.nonEmpty))
+        Success
+      else
+        RuleViolation(jobRunSpec, JobRunSpecMessages.cmdOrDockerValidation, None)
+    }
+  }
+}
+
+object JobRunSpecMessages {
+  val cmdOrDockerValidation = "Cmd or docker image must be specified"
```

Comment:
nit: `cmd` or `Command` instead of `Cmd`?

Reply:
Since this is only backport I guess we can live with this :)